### PR TITLE
Support Red Hat Disabled rules (aka autoack rules in the API)

### PR DESCRIPTION
**Pull request by mhuth**
Seems to work ok with the API changes in PR: https://github.com/RedHatInsights/insights-advisor-api/pull/642.  But they will need to go into prod together, otherwise :scream:

### CODE REVIEW

**Comment by mhuth** [comment]
Showing all rules (using reports_shown === 'all' or just excluding the reports_shown parameter)
all_rules_list

Showing only the list of Red Hat Disabled rules:
redhat_disabled_rule_list

Showing the Red Hat Disabled rule detail:
redhat_disabled_rule_detail

Note that enabling the rule worked correctly too.  Here's the Red Hat Disabled rule enabled again:

File: src/SmartComponents/Recs/List.js [hunk 1]
```diff
@@ -13,15 +15,32 @@ import { isGlobalFilter } from '../../AppConstants';
 const RulesTable = lazy(() => import(/* webpackChunkName: "RulesTable" */ '../../PresentationalComponents/RulesTable/RulesTable'));
 const TagsToolbar = lazy(() => import(/* webpackChunkName: "TagsToolbar" */ '../../PresentationalComponents/TagsToolbar/TagsToolbar'));
 
+let AdvisorRedHatDisabledRuleAlert = sessionStorage.getItem('AdvisorRedHatDisabledRuleAlert') || true;
```
Comment (AllenBW) [inline src/SmartComponents/Recs/List.js:18]: OH GOOD I WAS WONDERING, cuz i got dinged last time for not having this 🤗🤗🤗🤗🤗

**Comment by AllenBW** [comment]
@mhuth @dkuc is the api change needed for this breaking? or can it go to prod n we'll merge this whenever, no issues 🤔

File: src/SmartComponents/Recs/List.js [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import './List.scss';
 
 import { PageHeader, PageHeaderTitle } from '@redhat-cloud-services/frontend-components/components/PageHeader';
-import React, { Suspense, lazy } from 'react';
+import React, { Suspense, lazy, useState } from 'react';
 
+import { Alert } from '@patternfly/react-core/dist/js/components/Alert/Alert';
```
Comment (AllenBW) [inline src/SmartComponents/Recs/List.js:6]: lets use the ems version of this, same for below

**Review by AllenBW — CHANGES_REQUESTED** [review]
Just the one thing... OO no and another thing... be sure to run 

`npm run translations` so we get all them new stringzzzzzzzzzzzzzz captured n such 🤗 

LOOKS AWESOME OTHERWISE :AWESOME: 👍 🧑‍🏭

**Comment by mhuth** [comment]
> @mhuth @dkuc is the api change needed for this breaking? or can it go to prod n we'll merge this whenever, no issues

@AllenBW The API change will break the frontend if they don't go into production together.  I guess I should change the API so it'll work with either version of the frontend, ie. using reports_shown=true or reports_shown=enabled.  I'll fix that up tomorrow ...

**Comment by terezanovotna** [comment]
Looking good!
 - the inline notification should not spread across full screen - should be as in [mocks](https://marvelapp.com/prototype/e5gcj47/screen/71274223) or [here is documentation](https://www.patternfly.org/v4/design-guidelines/usage-and-behavior/alerts-and-notifications)
- Red Hat disabled (no capitalization of D)

**Comment by mhuth** [comment]
Going to close this in favour of PR https://github.com/RedHatInsights/insights-advisor-frontend/pull/757

**Pull request closed without merging**